Library/Homebrew/os/mac/sdk.rb

```diff
@@ -28,17 +28,32 @@ def latest_sdk
         SDK.new v, path
       end
 
+      def sdk_if_applicable(v = nil)
+        begin
+          sdk = if v.nil?
+            (source_version.to_i >= 7) ? latest_sdk : sdk_for(OS::Mac.version)
```

Comment:
Maybe make this a block `if` or `elsif` if that looks cleaner?

Reply:
I can make those changes, but this code hasn't actually changed - this is the exact same method that used to be in `MacOS.sdk`. It's just moved.